tmt/steps/provision/podman.py

```diff
@@ -157,19 +157,20 @@ def ansible(self, playbook, extra_args=None):
         playbook = self._ansible_playbook_path(playbook)
         # As non-root we must run with podman unshare
         podman_unshare = 'podman unshare ' if os.geteuid() != 0 else ''
+        self.run(f'stty cols {tmt.utils.OUTPUT_WIDTH};:',
+                 shell=True)
         stdout, stderr = self.run(
-            f'stty cols {tmt.utils.OUTPUT_WIDTH}; '
-            f'{self._export_environment()}'
             f'{podman_unshare}ansible-playbook '
             f'{self._ansible_verbosity()} '
             f'{self._ansible_extra_args(extra_args)} '
-            f'-c podman -i {self.container}, {playbook}',
-            cwd=self.parent.plan.worktree)
+            f'-c podman -i {self.container}, {playbook}'.split(),
```

Comment:
Is split necessary? One-time manual split into list items is safer (there might be space in playbook path)